docs/DeveloperGuide.md

```diff
@@ -834,3 +880,21 @@ testers are expected to do more *exploratory* testing.
        The terminal from where `WedLinker.jar` is launched should log where the file is corrupted.
 
     3. If the data is beyond repair, delete the entire `docs` folder and the `AddressBook.json` file to start afresh.
+
+
+## Appendix: Planned Enhancements
+
+Team size: 5
+
+Based on the current implementation of WedLinker, there are known bugs and limitations that we are unable to resolve due to
+feature freeze. The plans to improve our features are as such.
+
+1. There is a small GUI bug that causes the overflow of names for Tags and Weddings which causes cosmetic flaws.
+Planned enhancements would be to truncate the name with `...`.
+2. There is a duplicate validation bug that allows the creation of duplicate Person, Wedding, Tag, Task, Tags.
+Planned enhancements would be to ensure parser strips all whitespace except one between the keywords before creating the
+respective command objects.
+3. There is a missing validation in unassign-task command that negates the check of whether a Person is a vendor, resulting 
+in an incorrect error message to be shown. However, there is no functionality flaws and application runs as intended.
```

Comment:
Can maybe add line after '... to be shown.' to say 'The error message indicates that there are no tasks in the person's list, rather than indicating that the person is not a Vendor.'